purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -173,6 +179,15 @@ open class PurchasesConfiguration(builder: Builder) {
             this.dangerousSettings = dangerousSettings
         }
 
+        /**
+         * Disable this setting if you don't want to allow pending purchases for prepaid subscriptions (only supported
+         * in Google Play). Note that entitlements are not granted until payment is done.
+         * Default is enabled.
+         */
+        fun pendingPrepaidSubscriptionsEnabled(pendingPrepaidSubscriptionsEnabled: Boolean) = apply {
```

Comment:
I noticed Google calls this feature "pending transactions for prepaid plans", e.g. [here](https://developer.android.com/google/play/billing/subscriptions#pending). What do you think about something like `pendingTransactionsForPrepaidPlansEnabled()`?

Reply:
+1. 

As a side note, if you say `pendingTransactionsForPrepaidPlansEnabled` 3 times in a row quickly I think that nets you an honorary Master's Degree in Diction

Reply:
Renamed in https://github.com/RevenueCat/purchases-android/pull/1717/commits/67a43a68ca9aad093b14cd717f942a85c4ede392